Makefile

```diff
@@ -48,7 +50,7 @@ docker-run-it:
 	docker run -it --rm -p 10000:10000 -p 11000:11000 -e SERVE_HTTP=true ${IMG}
 
 build:
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o ./bin/app
+	CGO_ENABLED=0 GOOS=${OS} GOARCH=${ARCH} go build -a -o ./bin/app
```

Comment:
I think we should abstain from doing this change, I don't know what's the infra behind the pipelines that build this nor the infra where this is deployed to. Also, `uname -m` returns `x86_64` not `amd64`.
I understand that this change was convenient for you to test it locally but I want to avoid breaking version service for the whole company.

Reply:
> It varies, depending on the system you run it on. 😅 For me it returns arm64, but it can even run amd64 or x86 (with rosetta).

I know :see_no_evil: I was trying to say that for x86 64-bit CPUs it doesn't return `amd64` which is the value that GOARCH expects, it returns `x86_64` which I believe would lead to an error in the VS CI pipeline.